### PR TITLE
BC - Added the endpoint for finalsInfo

### PR DESCRIPTION
**Pull request by Bennyyyyy**
Finished part 2 of Finals Information
- Added the endpoint
- Added some basic tests to test functionality
- Returns for the data query f23 and 12345

Dokku: https://proj-courses-bennyyyyy.dokku-12.cs.ucsb.edu/
image

Closes: #59

### CODE REVIEW

**Review by anisavic — APPROVED** [review]
LGTM

File: src/main/java/edu/ucsb/cs156/courses/controllers/FinalsInfoController.java [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+          @RequestParam
+          String quarter,
+      @Parameter(
+              name = "enrollCd",
+              description = "Enrollment code for the course",
+              example = "12345",
+              required = true)
+          @RequestParam
+          String enrollCd) {
+    // FinalsInfo finalsInfo = getFinalsInfo(quarter, enrollCd)
```
Comment (GlassBricks) [inline src/main/java/edu/ucsb/cs156/courses/controllers/FinalsInfoController.java:42]: Is this commented out code supposed to be in this PR?

If you plan on "finishing" the functionality in a following PR, please state so (and you can then ignore this comment)

**Review by GlassBricks — CHANGES_REQUESTED** [review]

**Comment by Bennyyyyy** [comment]
I removed the comments. The functionality is to be implemented later on.

**Review by audoreven — DISMISSED** [review]
LGTM

**Pull request closed without merging**